packages/hooks/__tests__/use-form-item.spec.ts

```diff
@@ -36,15 +36,15 @@ const mountComponent = (setup = NOOP, options = {}) => {
 
 const getButtonVm = (wrapper: ReturnType<typeof mountComponent>) => {
   return wrapper.findComponent(ElButton).vm as any as {
```

Comment:
**Question:** why do we need to cast the type here? Shouldn't `ElButton` be type annotated already?

Reply:
Exported variables are not attached to the Button type.

Reply:
Just as you mentioned it is not attached, should we try test the variable via other options like in the code, `_size` is used here: `n.m(_size)`, shouldn't we test this by finding whether the variable exists or not? For instance:

```ts
expect(wrapper.find('el-button--default').exists()).toBeTrue()
```